resources/diameter-s13/pom.xml

```diff
@@ -0,0 +1,122 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <packaging>pom</packaging>
+
+    <parent>
+        <groupId>org.mobicents</groupId>
+        <artifactId>mobicents-parent</artifactId>
+        <version>2.21</version>
+    </parent>
+
+    <groupId>org.mobicents.servers.diameter.resources</groupId>
+    <artifactId>s13-parent</artifactId>
+    <version>1.0.0.FINAL</version>
+
+    <name>Mobicents :: Diameter Resources :: ${pom.artifactId}</name>
+
+    <modules>
+        <module>ratype</module>
+        <module>events</module>
+        <module>ra</module>
+        <module>du</module>
+    </modules>
+    <developers>
+        <developer>
+            <id>PCB</id>
+            <name>Paul Carter-Brown</name>
+            <email>paul.carter-brown@smilecoms.com</email>
+            <organization>Smile Communications</organization>
+            <organizationUrl>http://www.smilecoms.com</organizationUrl>
+        </developer>
+    </developers>
+
+    <properties>
+        <!-- Mobicents Diameter Components Versions -->
+        <mobicents.diameter.resources.base-common.version>2.7.0.FINAL</mobicents.diameter.resources.base-common.version>
```

Comment:
Please remove this, we use project.version for org.mobicents.resources dependencies.